src/components/Filters/index.js

```diff
@@ -361,6 +366,7 @@ class Filters extends React.Component {
               saveView={(name, scope) => this.saveView(name, scope)}
               rules={rules}
               views={this.props.views || []}
+              schema={this.props.schema}
```

Comment:
This should be also passed to the ViewsMenu component (so that we can save an restore views).